### ISSUE TITLE
Questionable log quality

### ISSUE THREAD

**Issue by JenaroG** [sac-water/neuralog#12]
at 2950 the log has been stretched. Would you still want me to digitize the whole log or skip the altered section? 
API# 02903969

**Comment by whdc** [comment]
Yes, let's skip 2940 thru 3030. (Note to self: the perf interval is 3517–5248.)
